nncf/experimental/common/pruning/operations.py

```diff
@@ -275,6 +284,26 @@ def _get_output_mask(cls, input_masks: List[Optional[PropagationMask]]) -> Optio
                             group.invalidate()
         return output_mask
 
+    @staticmethod
+    def can_propagate_mask(input_masks: List[Optional[PropagationMask]], input_shapes: List[Tuple[int, ...]]) -> bool:
+        """
+        Check that input without a mask does not affect input with a mask.
+
+        :param input_masks: Input masks.
```

Comment:
please make it more descriptive
e.g. list of propagation masks for each input of the element-wise operation

Reply:
done